src/client/theme-default/components/VPNavBarMenu.vue

```diff
@@ -24,6 +24,8 @@ const { theme } = useData()
 @media (min-width: 768px) {
   .VPNavBarMenu {
     display: flex;
+    align-items: center;
+    padding-inline-start: 12px;
```

Comment:
The NavBarMenu didn't vertically center its children. Instead, all its children had a `height: var(--vp-nav-height-mobile)`. When changing those children heights to `min-height` instead, some children can end up being taller than others, so we do need `align-items` as well.

The `padding-inline-start` is to prevent menu items from ending up _very_ close to the Search button (or to the sidebar when there is no Search button).